### PR TITLE
Sync Strings

### PR DESCRIPTION
**Pull request by ElementBot**
- Update Strings from Localazy

### CODE REVIEW

**Review by julioromano — APPROVED** [review]

File: features/messages/impl/src/main/res/values-fr/translations.xml [hunk 1]
```diff
@@ -4,10 +4,27 @@
     <item quantity="one">"%1$d changement dans la conversation"</item>
     <item quantity="other">"%1$d changements dans la conversation"</item>
   </plurals>
+  <plurals name="screen_room_timeline_more_reactions">
+    <item quantity="one"></item>
```
Comment (bmarty) [inline features/messages/impl/src/main/res/values-fr/translations.xml:8]: There is a pb here, the value is missing. The CI should detect missing values. :/

File: features/messages/impl/src/main/res/values-sk/translations.xml [hunk 2]
```diff
@@ -5,6 +5,11 @@
     <item quantity="few">"%1$d zmeny miestnosti"</item>
     <item quantity="other">"%1$d zmien miestnosti"</item>
   </plurals>
+  <plurals name="screen_room_timeline_more_reactions">
+    <item quantity="one"></item>
```
Comment (bmarty) [inline features/messages/impl/src/main/res/values-sk/translations.xml:9]: There is a pb here, the value is missing. The CI should detect missing values. :/

File: features/messages/impl/src/main/res/values-fr/translations.xml (hunk 1 above)
Comment (julioromano) [reply features/messages/impl/src/main/res/values-fr/translations.xml:8]: It does but it's only a warning so I thought it was intentional

Comment (bmarty) [reply features/messages/impl/src/main/res/values-fr/translations.xml:8]: I have created https://github.com/vector-im/element-x-android/issues/885 to improve that when we will have more time.

**Pull request merged**